src/main/app/pdf/mappers/theirDetailsMapper.ts

```diff
@@ -21,6 +21,23 @@ export class TheirDetailsMapper {
     return data
   }
 
+  static partyTypeAsString (partyDetails: TheirDetails): string {
+    if (partyDetails && partyDetails.type) {
+      switch (partyDetails.type) {
+        case PartyType.INDIVIDUAL.value:
```

Comment:
This is just pointless. Since all party types have `displayValue` property it does not make any sense to have switch statement. Much easier would be to do `return partyDetails.type.displayValue`